.github/workflows/e2e-tests.yml

```diff
@@ -6,16 +6,21 @@ on:
   pull_request:
 jobs:
   e2e-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        parallel-id: [0, 1, 2, 3]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
           go-version: '~1.16'
-      - run: make e2e-local E2E_NODES=2 ARTIFACTS_DIR=./artifacts/
+      - run: make e2e-local E2E_TEST_CHUNK=${{ matrix.parallel-id }} E2E_TEST_NUM_CHUNKS=${{ strategy.job-total }} E2E_NODES=2 ARTIFACTS_DIR=./artifacts/
       - name: Archive Test Artifacts # test results, failed or not, are always uploaded.
         if: ${{ always() }}
         uses: actions/upload-artifact@v2
         with:
-          name: e2e-test-output-${{(github.event.pull_request.head.sha||github.sha)}}-${{ github.run_id }}
+          name: e2e-test-output-${{ (github.event.pull_request.head.sha || github.sha) }}-${{ github.run_id }}-${{ matrix.parallel-id }}
           path: ${{ github.workspace }}/bin/artifacts/*
+# TODO: create job to combine test artifacts using code in https://github.com/operator-framework/operator-lifecycle-manager/pull/1476
```

Comment:
:+1:

Reply:
@njhale This might be a problem with some of the artifact parsing we've been talking about.

Reply:
Actually can we create a follow-up issue that tackles this? I don't think we need to block this PR merging, but I don't want to lose touch of this work.

Reply:
I'll create a follow-up ticket that tracks this work after this merges. We can add this to the #2571 tracking issue.